permits/tests/test_c_admin_site.py

```diff
@@ -249,3 +286,62 @@ def test_cannot_add_a_new_integrator_group_to_an_user_who_has_already_an_integra
             error_msg=admin.MULTIPLE_INTEGRATOR_ERROR_MESSAGE,
         )
         self.assertInHTML(expected, content)
+
+    if settings.ENABLE_2FA:
+
+        def test_user_of_group_with_mandatory_2FA_is_asked_to_set_it_up(self):
+            user = SecretariatUserFactory()
+            user.groups.set([self.group2fa])
+            self.client.login(username=user.username, password="password")
+
+            response = self.client.get(
+                reverse(settings.LOGIN_REDIRECT_URL), follow=True,
+            )
```

Comment:
I think it's good, but the user might choose another redirect with the "next" argument or by entering any path in the URL bar after the redirection. In this case, shouldn't he be enforced to use 2FA as well? (Related to a comment below)

Reply:
Solved by the decorator check